docker/Dockerfile

```diff
@@ -54,6 +54,7 @@ LABEL org.label-schema.docker.cmd 'docker run \
 RUN apt-get update && \
     apt-get install -y --no-install-recommends \
         apt-transport-https \
+        apt-utils \
```

Comment:
also add `Recommends: apt-utils` to https://salsa.debian.org/go-team/packages/debos/-/blob/master/debian/control